packages/application-shell/src/components/quick-access/create-commands.ts

```diff
@@ -1,18 +1,33 @@
+import { IntlShape } from 'react-intl';
 import { oneLineTrim } from 'common-tags';
 import { hasSomePermissions } from '@commercetools-frontend/permissions';
 import {
   LOGOUT_REASONS,
   SUPPORT_PORTAL_URL,
 } from '@commercetools-frontend/constants';
+import { TApplicationContext } from '@commercetools-frontend/application-shell-connectors';
 import { permissions } from './constants';
 import messages from './messages';
+import { Command } from './types';
 
-export default function createCommands({
-  applicationContext,
+function nonNullable<T>(value: T | boolean): value is NonNullable<T> {
+  return value !== null && value !== undefined && typeof value !== 'boolean';
+}
+
+type CreateCommandsOptions = {
+  intl: IntlShape;
+  applicationContext: TApplicationContext<{}>;
+  featureToggles: { [key: string]: boolean };
+  changeProjectDataLocale: (locale: string) => void;
+};
+
+const createCommands = ({
   intl,
+  applicationContext,
   featureToggles,
   changeProjectDataLocale,
-}) {
+}: CreateCommandsOptions): Command[] => {
+  console.log('featureToggles', featureToggles);
```

Comment:
Forgot to remove this. However, there seem to be something wrong with the feature toggles in the tests, I need to investigate this a bit.